sdk/eventhub/eventhubs-checkpointstore-blob/changelog.md

```diff
@@ -1,6 +1,14 @@
+### 2019-11-05 - 1.0.0-preview.5
+
+- Updated to use the latest version of the `@azure/event-hubs` package.
+
+Breaking changes:
+- `BlobPartitionManager` has been renamed to `BlobCheckpointStore` to reflect naming changes 
```

Comment:
Also add the part about `Checkpoint` and `PartitionOwnership` interfaces have changed

Reply:
And about the new method `listCheckpoints`?

Reply:
And about `updateCheckpoint` not returning a string?